newa/cli.py

```diff
@@ -590,12 +591,46 @@ def _jira_fake_id_generator() -> Generator[str, int, None]:
 
                 ctx.logger.info(f"Processing {action.id}")
 
+                # check if there are iteration is defined
+                if action.iterate:
+                    i = 0
+                    # for each value prepare a separate action
+                    for iter_vars in action.iterate:
```

Comment:
How about? And remove `i = 0` and `i += 1` lines..
```suggestion
                    for i, iter_vars in enumerate(action.iterate):
```

Reply:
I didn't know this one. Thanks!